### DISCUSSION TITLE
【Step-0】请各团队的队长注意！

### DISCUSSION THREAD

**Discussion by GreyZeng** [FZUSESPR21/Class_Resources#1] [General]
请各团队的队长注意！

请各团队的**队长**将自己的

- **Github地址**
- **学号**
- **团队名**

留言在此贴下。

截止时间：2021/03/24 晚上八点
逾期不候

**Comment by hb594709591** [comment]
Github地址：[我的github地址](https://github.com/hb594709591)
学号: 221801410
团队名: 青青草原

**Comment by gz321** [comment]
Github地址：https://github.com/gz321
学号：221801205
团队名：饱满骑士

**Comment by Lyr-GW** [comment]
Github地址：https://github.com/Lyr-GW
学号：221801314
团队名：烤盐人

**Comment by fzu-kwz** [comment]
Github地址：https://github.com/fzu-kwz
学号：221801111
团队名：老九门

**Comment by Tozzger** [comment]
Github地址：https://github.com/Tozzger
学号：041801114
团队名：福大周润发队

**Comment by Fzuerzmj** [comment]
Github地址：https://github.com/Fzuerzmj
学号：221801222
团队名：逐梦校友圈

**Comment by Honghonglin** [comment]
Github地址：https://github.com/Honghonglin
学号：221801335
团队名：疯狂的TuT们

**Reply by GreyZeng** [reply]
请提供队长的个人Github地址，不要提供组织地址。

**Comment by Hao23333** [comment]
Github地址：[https://github.com/Scott233](https://github.com/Scott233)
学号：221801101
团队名：字节乱动

**Comment by LinxuanZheng** [comment]
Github地址：https://github.com/ErrorPay
学号：221801218
团队名：评了么
